traffic_portal/test/integration/Data/servercapabilities.ts

```diff
@@ -47,67 +48,75 @@ export const serverCapabilities = {
 			],
 			check: [
 				{
-					description: "check CSV link from Server Capabilities page",
+					caseDescription: "check CSV link from Server Capabilities page",
 					Name: "Export as CSV"
 				}
 			],
 			add: [
 				{
-					description: "can create a server capability",
+					caseDescription: "can create a server capability",
 					name: "TP_SC",
+					description: "Server Capability",
```

Comment:
These descriptions aren't being used in the spec. The add cases should be setting the descriptions from these values.

Reply:
Oops.. I forgot to add it.. thanks for catching it.